### DISCUSSION TITLE
Frontend text only and admin panel is wrong link

### DISCUSSION THREAD

**Discussion by defactomail** [YOURLS/YOURLS#3324] [Q&A] [answered]
Hi. I got YOURLS to work properly (almost) on my VPS, but I'm having small problems I don't know how to fix.

**First**, the front end has no graphics or lay out. It looks like this:
![YOURLS](https://user-images.githubusercontent.com/102940616/166924360-9a7934a5-7912-4195-899f-1001f299781a.png)

**Second**, the "Admin Interface" link is not pointing to the correct sub-folder, which is "shop". Instead, it directs me directly to my wp-admin backend. So, it should be _https://domain.com/shop/admin/index.php_, but instead it's _https://domain.com/admin/index.php_...

Also, I turned on debuging and reloaded the YOURLS frontend, and this is what it gave me... I have no idea what the glitch is. I can't see any errors... (is it permission errors? Is so, what to do?)

- Connected to database defa_links on localhost 
SQL perform: SELECT option_name, option_value FROM yourls_options WHERE 1=1 (0.00008 s)
Check for new version: no
SQL perform: SELECT COUNT(keyword) as count, SUM(clicks) as sum FROM `yourls_url` WHERE 1=1  (0.00004 s)
SQL perform: SELECT * FROM `yourls_url` WHERE 1=1  ORDER BY `timestamp` desc LIMIT 0, 15; (0.00007 s)

How can I fix this?
Thanks

**Comment by dgw** [comment, marked as answer]
Make sure your webserver is configured correctly for YOURLS in a subdirectory (e.g. [for Apache, like this](https://docs.yourls.org/guide/server-configuration.html#make-a-htaccess-file)) and that `YOURLS_SITE` in your `config.php` points to the location of YOURLS (`https://domain.com/shop`).

**Reply by defactomail** [reply]
Thanks a ton. I had it pointing toward _https://domain.com_ instead of _https://domain.com/shop._

Problem fixed!
